fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/mapper/ManualMapper.java

```diff
@@ -0,0 +1,27 @@
+package de.adorsys.opba.fintech.impl.mapper;
+
+import de.adorsys.opba.fintech.api.model.generated.AccountList;
+import de.adorsys.opba.fintech.api.model.generated.BankDescriptor;
+import de.adorsys.opba.fintech.api.model.generated.BankProfile;
+import de.adorsys.opba.fintech.api.model.generated.TransactionsResponse;
+import de.adorsys.opba.tpp.banksearch.api.model.generated.BankProfileDescriptor;
+import org.mapstruct.factory.Mappers;
+
+@SuppressWarnings("checkstyle:HideUtilityClassConstructor")
+public class ManualMapper {
```

Comment:
What is the purpose of this class?

Reply:
To gather all automatic wrappers in one place. So I dont have to inject all of them.